controllers/cron_rollout_helpers.go

```diff
@@ -120,13 +119,18 @@ func createBaseDisruption(owner metav1.Object, disruptionSpec *chaosv1beta1.Disr
 }
 
 // setDisruptionAnnotations updates the annotations of a given Disruption object with those of its owner.
-// Additionally, it sets a scheduled time annotation using the provided scheduledTime.
-func setDisruptionAnnotations(disruption *chaosv1beta1.Disruption, owner metav1.Object, scheduledTime time.Time) {
-	for k, v := range owner.GetAnnotations() {
-		disruption.Annotations[k] = v
+// It sets a scheduled time annotation using the provided scheduledTime.
+// It parses the UserInfo annotation if it exists and sets user-related annotations.
+func setDisruptionAnnotations(disruption *chaosv1beta1.Disruption, owner metav1.Object, scheduledTime time.Time) error {
+	disruption.CopyOwnerAnnotations(owner)
+
+	disruption.SetScheduledAtAnnotation(scheduledTime)
+
+	if err := disruption.CopyUserInfoToAnnotations(owner); err != nil {
```

Comment:
I'm not sure how the linter feels about this, but you _could make this one line, `return disruption.CopyUserInfoToAnnotations(owner)`, since we just directly return `err` either way

Reply:
There is only one way to find out 😄

Reply:
All good 👍